lib/mongoid/interceptable.rb

```diff
@@ -152,6 +152,34 @@ def _mongoid_run_child_callbacks(kind, children: nil, &block)
       end
     end
 
+    # This is used to store callbacks to be executed later. A good use case for
```

Comment:
I think it would be helpful to explain "later" here; I believe what is meant is "later than according to standard callback order", but I am not sure.

Reply:
I don't think it is after than according to standard callback order. The thing is, I meant this as more of a general operator and not for a specific use case so it's hard to say what later is. That's why I included an example. But in that example, we're just waiting for the associations to get set, the callbacks should still be called in the correct order. Maybe actually the after_find/after_initialize now gets called after the associations after_add.

Reply:
:ok_man: